### PR TITLE
Introduce wait function and some tweaks

### PR DESCRIPTION
**Pull request by minvrs**

### CODE REVIEW

File: web/pageObjects/landing_page.ts [hunk 1]
```diff
@@ -2,7 +2,16 @@ import * as defaultPage from "./default_page"
...
+//----------Actions----------//
+
+export async function clickBtnSingInWithEmail(): Promise<void> {
     await defaultPage.clickByLocator(btnSignInWithEmail)
 
+}
+
+//----------Waits----------//
+
+export async function waitUntilBtnSingInWithEmail(): Promise<void> {
```
Comment (eglevens) [inline web/pageObjects/landing_page.ts]: unify function names 
waitUntilBtnSortQuizesIsVisible() vs waitUntilBtnSingInWithEmail() 
need to use `isVisible` everywhere or do not use it at all

File: web/pageObjects/default_page.ts [hunk 2]
```diff
@@ -10,23 +12,43 @@ export async function openSigInPageURL(): Promise<void> {
...
     await clickByLocator(locator)
     await (await getElementsByLocator(locator)).keys(text)
 
+}
+
+//----------Waits----------//
+
+let defaultTimeuot: number = 10000
+
+export async function waitUntilElemantIsVisibleInViewportByLocator(locator: string, customTimeout?: number): Promise<void> {
```
Comment (eglevens) [inline web/pageObjects/default_page.ts]: mistype **Elemant**

File: web/pageObjects/landing_page.ts (hunk 1 above)
Comment (minvrs) [reply web/pageObjects/landing_page.ts]: Good point, nesupdantu pats kodel nepridejau `isVisible`

File: web/pageObjects/default_page.ts (hunk 2 above)
Comment (minvrs) [reply web/pageObjects/default_page.ts]: fixed

**Comment by eglevens** [comment]
well done!

**Pull request merged**